docs/what-can-be-faked.md

```diff
@@ -35,3 +43,7 @@ overridden if they are:
 
 Note that this means that static members, including extension methods,
 **cannot** be overridden.
+
+### Methods with reference return values
+
+Methods with [reference return values](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/ref-returns#what-is-a-reference-return-value) can not be invoked on a Fake. Any attempt to do so will result in a `NullReferenceException` being thrown. 
```

Comment:
Even though the official documentation uses it, I find the term "reference return value" confusing... It could easily be understood as "return value of reference type". I don't have a good alternative besides "ref return", though... Any better idea?

Reply:
"methods that return values by reference"?

Reply:
Still confusing?

Reply:
Maybe just "Methods that return by reference"?

Reply:
Is okay with me. keep "(officially…)", though?

Reply:
Sorry, I posted my suggestion before I saw your answer. Either "methods that return values by reference" or "Methods that return a reference to a variable" are fine by me. Although the reference might be to an array location, rather than a variable...
Mentioning the official term is a good idea.